src/main/app/claims/models/response/core/responseAcceptance.ts

```diff
@@ -0,0 +1,10 @@
+import { PaymentIntention } from 'claims/models/response/core/paymentIntention'
+import { CourtDetermination } from 'claims/models/response/core/courtDetermination'
+
+export class ResponseAcceptance {
```

Comment:
Neither ResponseAcceptance nor ResponseRejection is used. Start using it either as a classes or interfaces.

Reply:
Both ResponseAcceptance and ResponseRejection should have type property as well to be valid.